app/main.py

```diff
@@ -17,3 +16,22 @@
 collection_of_coins = {1, 2, 25}
 
 # write your code here
+def sorted_variables(*args) -> dict:
+    variable_name = ["lucky_number", "pi", "one_is_a_prime_number", "name", "my_favourite_films", "profile_info",
+                     "marks", "collection_of_coins"]
+    variable_name_dict = dict(zip(variable_name, args))
+    immutable = []
+    mutable = []
+    result_dic = {}
+    for k, v in variable_name_dict.items():
+        if isinstance(v, (int, float, str, bool, tuple, type(None))):
+            immutable.append(k)
+            result_dic["mutable"] = mutable
+        else:
+            mutable.append(k)
+            result_dic["immutable"] = immutable
+    return result_dic
+
+
+
+print(sorted_variables(lucky_number, pi, one_is_a_prime_number, name, my_favourite_films,profile_info,marks, collection_of_coins))
```

Comment:
you made it difficult, do as it is written in README.md
<img width="659" alt="Screenshot 2023-12-06 at 20 59 43" src="https://github.com/mate-academy/py-mutable-immutable/assets/93856423/6b06ab7c-e9e2-4f76-8df7-dfc1627154f2">